INCode/models.py

```diff
@@ -196,19 +185,21 @@ def _is_a_call(cursor):
     def _is_a_callable(cursor):
         return cursor.kind == CursorKind.FUNCTION_DECL or cursor.kind == CursorKind.CXX_METHOD
 
-    def _get_class(self, cursor):
-        if cursor.kind == CursorKind.CXX_METHOD:
-            return cursor.lexical_parent.displayname
-        return '{} is not supported'.format(cursor.kind)
-
-    def _get_name(self, cursor):
-        if cursor.kind == CursorKind.FUNCTION_DECL or cursor.kind == CursorKind.CXX_METHOD:
-            return _get_function_signature(cursor)
-        return '{} is not supported'.format(cursor.kind)
-
-    def _get_sender(self, cursor):
-        if cursor.kind == CursorKind.FUNCTION_DECL:
-            return self.get_translation_unit()
-        elif cursor.kind == CursorKind.CXX_METHOD:
-            return self._get_class(cursor)
-        return '{} is not supported'.format(cursor.kind)
+    def _get_class(self):
+        if self.cursor_.kind == CursorKind.CXX_METHOD:
+            return self.cursor_.semantic_parent.displayname
+        return '{} is not supported'.format(self.cursor_.kind)
+
+    def _get_name(self, get_function_signature=False):
+        if self.cursor_.kind == CursorKind.FUNCTION_DECL or get_function_signature:
```

Comment:
What's the purpose of `or get_function_signature`?

Reply:
Will rename to `name_only`.